javascript/javascript1/week3/week3.js

```diff
@@ -0,0 +1,135 @@
+//username on freecodecamp: Rikkaa
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+const shiftName = names.shift(function (e) {
```

Comment:
Please have another look on this one. If you test your code you can see that Ahmad is still in the names array. 
.shift() is used to removed the first item of an array and does not take any argument.  (try using splice or filter instead) 🐱‍👓
also you are creating a variable shiftName and then not using it, that's not the best practice. 🤠